github/ci/services/prometheus-stack/patches/stack/production-control-plane/prometheus.yaml

```diff
@@ -14,3 +14,12 @@ spec:
     limits:
       cpu: 1
       memory: 12Gi
+  additionalScrapeConfigs:
```

Comment:
Where does this change come from, aren't we just moving the file?

Reply:
This additional scrap config was in the original kube-prometheus-stack.yaml files
https://github.com/kubevirt/project-infra/blob/main/github/ci/services/prometheus-stack/manifests/stack/common/kube-prometheus-stack.yaml#L1239

Since we do not need it in the performance cluster, I moved this to patch in the other clusters